test/e2e/revision_test.go

```diff
@@ -91,7 +91,7 @@ func revisionListWithService(r *test.KnRunResultCollector, serviceNames ...strin
 }
 
 func revisionDelete(r *test.KnRunResultCollector, revName string) {
-	out := r.KnTest().Kn().Run("revision", "delete", revName)
+	out := r.KnTest().Kn().Run("revision", "delete", "--no-wait=false", revName)
```

Comment:
Wasn't there a `--wait` option, too ?

Reply:
At least I thouhgt that was what we have decided to use for boolean flags (`--bla` and `--no-bla` ...)

Reply:
No. 
The flag isnt using `flags.AddBothBoolFlagsUnhidden` which auto defines the other flag.
I think we'd need to fix this to use ^^ util for this bool flag definition.

Reply:
Using `--wait` flag now.